analysis_options.yaml

```diff
@@ -46,10 +46,10 @@ linter:
     - prefer_collection_literals
     - prefer_const_constructors
     - prefer_contains
-    - prefer_expression_function_bodies
+    # - prefer_expression_function_bodies
     # - prefer_final_fields
     # - prefer_final_locals
-    - prefer_function_declarations_over_variables
+    # - prefer_function_declarations_over_variables
```

Comment:
To get rid of noisy lints that we have no intention of addressing.

If someone wants to address them, thats cool - but the analyzer view within the IDE is most useful when it is empty when you start your work.